pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetGovernedStudyParticipantsRoute.java

```diff
@@ -32,14 +33,16 @@ public Object handle(Request request, Response response) {
 
```

Comment:
Can you add route tests for the new routes and the route changes here? They should be part of the `IntegrationTestSuite`.

Reply:
Test created in 99bc846